oshdb-tool/etl/src/main/java/org/heigit/ohsome/oshdb/tool/importer/util/OSHDbToIgnite.java

```diff
@@ -29,42 +30,39 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-public class OSHDB2Ignite {
+public class OSHDbToIgnite {
```

Comment:
why `OSHDbToIgnite` and not `OSHDBToIgnite`, since we fully capitalize `OSHDB` everywhere else?